### ISSUE TITLE
QA tests should use real string-values, instead of hard-coded values

### ISSUE THREAD

**Issue by DinisCruz** [TeamMentor/TM_4_0_Design#267]
For example 
https://github.com/TeamMentor/TM_4_0_QA/blob/master/tests/jade/pages-anonymous.test.coffee#L71-L88
![image](https://cloud.githubusercontent.com/assets/656739/5788434/258f02a6-9e2d-11e4-8b05-146449bed7a4.png)

... should be using the transformation of the respective markdown content , for example https://github.com/TeamMentor/TM_4_0_Design/blob/master/source/content/landing-pages/selling-point-1.md

![image](https://cloud.githubusercontent.com/assets/656739/5788436/5133da30-9e2d-11e4-81a2-4dcced8a5b79.png)

... which exist in the TM_4_0_Design repo

So we will need something like https://github.com/TeamMentor/TM_4_0_Design/issues/266

Related to https://github.com/TeamMentor/TM_4_0_Design/issues/265

**Comment by michaelhidalgo** [comment]
Yeah I agree, we should also come up with a name convention, for instance expectedUrl, expectedLoginPage. The test will become even easier to read.

**Comment by DinisCruz** [comment]
correct, 

for example we could name those variables similar to these [Jade-API](https://github.com/TeamMentor/TM_4_0_QA/blob/master/tests/API/Jade-API.coffee#L42-L63) methods

![image](https://cloud.githubusercontent.com/assets/656739/5789091/06f72a7c-9e54-11e4-966a-ae7828ce05d8.png)
